Src/SDK/Common/Temporal.Serialization/public/IPayloadCodec.cs

```diff
@@ -1,12 +1,12 @@
-﻿using System.Threading;
-using System.Threading.Tasks;

```

Comment:
Oh, I think the line endings slipped through here as well. :)

Reply:
I got this one.